watcher/src/watcher_db.rs

```diff
@@ -1846,170 +1837,152 @@ pub mod tests {
     // Verification report polling queue should behave as expected.
     #[test_with_logger]
     fn test_remove_all_for_source_url(logger: Logger) {
```

Comment:
This test change looks significant because of the indentation change; I find it easier to review with the "Ignore whitespace" option: https://github.com/mobilecoinfoundation/mobilecoin/pull/2205/files?diff=split&w=1